symfony/messenger/6.0/config/packages/messenger.yaml

```diff
@@ -12,3 +12,9 @@ framework:
         routing:
             # Route your messages to the transports
             # 'App\Message\YourMessage': async
+
+# when@test:
+#    framework:
+#        messenger:
+#            transports:
+#                async: 'in-memory://' # replace your transport name here
```

Comment:
I'd move the comment onto the previous line, otherwise it kind of looks like you're saying that the user should change `in-memory` to something else.

Unrelated, I'd like to add a link to https://github.com/zenstruck/messenger-test also. Maybe:

```yml
# For more Messenger testing tools, see https://github.com/zenstruck/messenger-test
```

Reply:
@weaverryan 👍 done. Thanks for your review.